hdf5.spec

```diff
@@ -283,7 +283,7 @@ do
   mkdir $mpi
   pushd $mpi
 %if (0%{?suse_version} >= 1500)
-  module load gnu-mpich/3.3
+  module load gnu-mpich
```

Comment:
https://github.com/daos-stack/hdf5/blob/86cfde917766e1eeb858117df507eef2d6c51eef/hdf5.spec#L320
and
https://github.com/daos-stack/hdf5/blob/86cfde917766e1eeb858117df507eef2d6c51eef/hdf5.spec#L398

I think that's it.  Maybe search for others to be sure though.

Reply:
Done